xrpl/transfer.js

```diff
@@ -0,0 +1,71 @@
+const xrpl = require('xrpl');
+const { Command, Option } = require('commander');
+const { getWallet, getAccountInfo, sendTransaction, hex } = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const { loadConfig, printInfo, printWarn, getChainConfig } = require('../common');
+
+async function processCommand(wallet, _, chain, args, options) {
```

Comment:
```suggestion
async function transfer(wallet, _, chain, args, options) {
```
name it after the command perhaps?